src/entry/type.ts

```diff
@@ -1,4 +1,4 @@
-export interface Entry {
```

Comment:
rename all `type` files to `types`

Reply:
actually let's have single `src/types.ts` and `src/interfaces.ts` where we will export all interfaces and types